website/src/components/ReviewPage/ReviewCard.tsx

```diff
@@ -128,19 +134,36 @@ const ButtonBar: FC<ButtonBarProps> = ({
     deleteAccessionVersion,
     editAccessionVersion,
     viewSequences,
+    viewFiles,
 }) => {
     const buttonBarClass = (disabled: boolean) =>
-        `${
-            disabled ? 'text-gray-300' : 'text-gray-500 hover:text-gray-900 hover:cursor-pointer'
-        } pl-3 inline-block mr-2 mb-2 text-xl`;
+        `${disabled ? 'text-gray-300' : 'text-gray-500 hover:text-gray-900 hover:cursor-pointer'} inline-block text-xl`;
     const approvable =
         sequenceEntryStatus.status === processedStatus &&
         !(sequenceEntryStatus.processingResult === errorsProcessingResult);
     const notProcessed = sequenceEntryStatus.status !== processedStatus;
 
     return (
         <div className='flex mb-auto pt-3.5 items-center'>
-            <div className='flex space-x-1'>
+            <div className='flex space-x-4'>
+                {viewFiles && (
+                    <>
+                        <button
+                            className={buttonBarClass(notProcessed)}
+                            onClick={viewFiles}
+                            data-tooltip-id={'view-files-tooltip' + sequenceEntryStatus.accession}
+                            data-testid={`view-files-${sequenceEntryStatus.accession}`}
+                            key={'view-files-button-' + sequenceEntryStatus.accession}
+                            disabled={notProcessed}
```

Comment:
All the checking of `notProcessed` here doesn't make sense as the button doesn't appear at all if `notProcessed`

Reply:
removed!